### PR TITLE
Update jakepaul.py

### PR DESCRIPTION
**Pull request by Electromaster232**
Will do tons of stuff

### CODE REVIEW

**Comment by DavidTechnoboy** [comment]
my name is  j e f f

File: jakepaul/jakepaul.py [hunk 1]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def fuckjakepaul (self):
-        
+        """FUCK HIM"""
         await self.bot.say("FAKE WIFE! FAKE LIFE! ( ͡° ͜ʖ ͡°) https://www.youtube.com/watch?v=tCMx_RPLCYc")
 		
     @commands.command()
     async def fuckjakepaulreact (self):
-        
+        """PEOPLE REACT TO: FUK JAKE PAUL!"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: it was jake paul himself

File: jakepaul/jakepaul.py [hunk 2]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def assulted (self):
-        
+        """ASSULTED"""
         await self.bot.say("it was more extreme ( ͡° ͜ʖ ͡°) https://youtu.be/DUiD7j6DdVQ?t=12m57s")
 		
     @commands.command()
     async def assultedparody (self):
-        
+        """PARODY"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: we had "its everyday bro parody" but you didnt call it a parody lololol

File: jakepaul/jakepaul.py [hunk 3]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def itseverydaybro (self):
-        
+        """IT'S EVERYDAY BRO"""
         await self.bot.say("ITS EVERYDAY BRO! ( ͡° ͜ʖ ͡°) https://www.youtube.com/watch?v=hSlb1ezRqfA")
 		
     @commands.command()
     async def itseverydaybroparody (self):
-        
+        """YO MY NAME IS NICK CROMPTON AND MY COLLAR STAY POPPIN"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: read lower comment

File: jakepaul/jakepaul.py [hunk 4]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def jakepaulthemesong (self):
-        
+        """THEME SONG BROTHA"""
         await self.bot.say("JAKE PAULERS KEEP IT LIT! ( ͡° ͜ʖ ͡°) https://www.youtube.com/watch?v=I0Se3ce433Q")
 	
     @commands.command()
     async def itseverynightsis (self):
-        
+        """EVERY NIGHT"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: you did not define a gender (sis, bro)

File: jakepaul/jakepaul.py [hunk 5]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def fuckjakepaulreact (self):
-        
+        """PEOPLE REACT TO: FUK JAKE PAUL!"""
         await self.bot.say("HACKED BY BANKS! ( ͡° ͜ʖ ͡°) https://www.youtube.com/watch?v=5RoNgAVP3K4")
 		
     @commands.command()
     async def jakepaulthemesong (self):
-        
+        """THEME SONG BROTHA"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: incorrect spelling of brother and as a mature person, i feel obligated to correct you.

File: jakepaul/jakepaul.py [hunk 6]
```diff
@@ -9,52 +9,52 @@ def __init__(self, bot):
...
     @commands.command()
     async def dabonhaters (self):
-        
+        """DAB ON THE HATERS"""
         await self.bot.say("DAB ON THE HATERS! ( ͡° ͜ʖ ͡°) https://imgur.com/a/CKsco")
 	
     @commands.command()
     async def jakepaulpray (self):
-        
+        """PRAY"""
```
Comment (DavidTechnoboy) [inline jakepaul/jakepaul.py]: god is dead

File: jakepaul/jakepaul.py (hunk 1 above)
Comment (DavidTechnoboy) [reply jakepaul/jakepaul.py]: incorrect spelling of fuck

**Comment by Electromaster232** [comment]
Alrighty then...

**Comment by DavidTechnoboy** [comment]
oh u can still comment

**Pull request closed without merging**